<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application</class>
 <widget class="QWidget" name="Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* *********************************************************** */

QWidget#Application {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(12, 12, 12, 255));
}

QWidget {
font-family: &quot;Century gotic&quot; ;
}

QPushButton, QGroupBox, QRadioButton, QComboBox, QSlider, QLCDNumber, QCheckBox, QSpinBox, QLabel, QProgressBar, QLineEdit, QWebView, QTextEdit, QDoubleSpinBox
{
border-width: 1px;
border-radius: 5px;
border-color: black;
border-style: outset;
color: white;
}


QPushButton
{
font: bold 30px;
}

QPushButton:on {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
}

QPushButton:!on{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
}


/* *********************************************************** */

QGroupBox {

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
font: bold 16px;
color: black;
margin-top: 1.5ex; /* leave space at the top for the title */
}

QGroupBox::title {
subcontrol-origin: margin;
subcontrol-position: top center; 
padding: 2 10px;                         
border-width: 1px;
border-radius: 5px;
border-color: black;
border-style: outset;
color: white;

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
 }

/* *********************************************************** */

QRadioButton {
border: none;
background: none;
}

 QRadioButton::indicator::unchecked {
     image: url(:/Images/Grey.png);
 }

 QRadioButton::indicator:unchecked:pressed {
     image: url(:/Images/Grey.png);
 }

 QRadioButton::indicator::checked {
     image: url(:/Images/Yellow.png);
 }

 QRadioButton::indicator:checked:pressed {
     image: url(:/Images/Yellow.png);
 }

/* *********************************************************** */



/* *********************************************************** */

QComboBox {

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
font: bold 16px;
padding: 1px 18px 1px 3px;

}


 QComboBox::down-arrow {
     image: url(:/Images/buttonDown.png);
 }


 QComboBox:on { /* shift the text when the popup opens */
     padding-top: 13px;
     padding-left: 14px;
 }

 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 30px;
     border-top-right-radius: 0px; 
     border-bottom-right-radius: 10px;
 }


QComboBox QAbstractItemView {
selection-background-color: lightgray;
background-color: grey;
background-clip: blue;
border-width: 2px;
border-radius: 0px;
border-color: black;
border-style: outset;
font: bold 16px;
color: white;
padding: 15px 4px;
 }


 QComboBox::down-arrow:on { /* shift the arrow when popup is open */
     top: 1px;
     left: 1px;
 }


QComboBox QScrollBar:vertical {
      border: 1px outset black;
	 background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(173, 173, 173, 255), 			stop:1 rgba(95, 95, 95, 255));
      width: 25px;
      margin: 22px 0 22px 0;
	border-radius: 2px;
padding: 5px 2px;
  }

QComboBox QScrollBar::handle:vertical {
      background: qlineargradient(spread:pad, x1:0, y1:0.034, x2:0, y2:1, stop:0.0282486 rgba(255, 212, 73, 255), stop:0.505682 rgba(255, 212, 73, 255), stop:0.517045 rgba(255, 194, 66, 255), stop:1 rgba(255, 141, 0, 255));
      min-height: 25px;
  }

QComboBox  QScrollBar::add-line:vertical {
      height: 25px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
background: none;
      margin: 2px 0 2px 0;
  }

QComboBox  QScrollBar::sub-line:vertical {

      height: 20px;
      subcontrol-position: top;
      subcontrol-origin: margin;
background: none;
      margin: 2px 0 2px 0;
  }


QComboBox  QScrollBar::up-arrow:vertical {
     image: url(:/Images/buttonUp.png);
  }
QComboBox QScrollBar::down-arrow:vertical {
     image: url(:/Images/buttonDown.png);
}

QComboBox  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }



/* *********************************************************** */
QSlider::groove:horizontal { 
border: 1px solid #999999;
height: 5px; 
background: rgb(52, 52, 52);
margin: 2px 2;
left: 8px;
right: 8px;
}

QSlider::handle:horizontal {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
border-width: 1px;
border-radius: 5px;
border-color: black;
border-style: outset;
width: 50px;
margin: -25px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}

QSlider::sub-page:horizontal {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 60, 255));
border: 1px solid #777;
border-radius: 2px;
border: 1px solid RGB(50,50,50);
}

QSlider::add-page:horizontal {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(12, 12, 12, 255));
border: 1px solid black;
border-radius: 2px;
}

QSlider {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
/*border-top-left-radius: 0px; 
border-bottom-left-radius: 0px; */
}



/* *********************************************************** */
QSlider::groove:vertical { 
border: 1px solid #999999;
width: 10px; 
background: rgb(52, 52, 52);
margin: 5 0 5 0px;
left: 8px;
right: 8px;
}

QSlider::handle:vertical {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
border-width: 1px;
border-radius: 5px;
border-color: black;
border-style: outset;
height: 50px;
margin:  0 -25; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */

}

QSlider::sub-page:vertical {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 60, 255));
border: 1px solid #777;
border-radius: 2px;
border: 1px solid RGB(50,50,50);
margin:  5 0 0 0px;
}

QSlider::add-page:vertical {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(12, 12, 12, 255));
border: 1px solid black;
border-radius: 2px;
margin:  0 0 5 0px;
}



/* *********************************************************** */

QLCDNumber {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
color: white;
text-align: center;

}


/* *********************************************************** */

QLabel {
background: none;
color: rgba(171,171,171,255);
border-style: none;
font: 14px;
}

/* *********************************************************** */

QSpinBox
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
font:  30px;
padding-left: 15px;
}

QSpinBox::up-button {
     subcontrol-origin: border;
     subcontrol-position: top right; /* position at the top right corner */
     width: 30px; 
     border-width: 1px;
     border-left-color: black;
     border-left-style: outset; 

 }
 QSpinBox::down-button {
     subcontrol-origin: border;
     subcontrol-position: bottom right; /* position at bottom right corner */

     width: 30px;
     border-width: 1px;
     border-left-color: black;
     border-left-style: outset; 
 }


 QSpinBox::up-arrow {
     image: url(:/Images/buttonUp.png);
     width: 25px;
     height: 25px;
 }
 QSpinBox::down-button:pressed {
     top: 1px;
     left: 1px;}
 QSpinBox::up-button:pressed {
     top: 1px;
     left: 1px;}
 QSpinBox::down-arrow {
     image: url(:/Images/buttonDown.png);
     width: 25px;
     height: 25px;
 }
/* *********************************************************** */

QDoubleSpinBox
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
font:  30px;
padding-left: 15px;
}

QDoubleSpinBox::up-button {
     subcontrol-origin: border;
     subcontrol-position: top right; /* position at the top right corner */
     width: 30px; 
     border-width: 1px;
     border-left-color: black;
     border-left-style: outset; 

 }
 QDoubleSpinBox::down-button {
     subcontrol-origin: border;
     subcontrol-position: bottom right; /* position at bottom right corner */

     width: 30px;
     border-width: 1px;
     border-left-color: black;
     border-left-style: outset; 
 }


 QDoubleSpinBox::up-arrow {
     image: url(:/Images/buttonUp.png);
     width: 25px;
     height: 25px;
 }
 QDoubleSpinBox::down-button:pressed {
     top: 1px;
     left: 1px;}
 QDoubleSpinBox::up-button:pressed {
     top: 1px;
     left: 1px;}
 QDoubleSpinBox::down-arrow {
     image: url(:/Images/buttonDown.png);
     width: 25px;
     height: 25px;
 }



/* *********************************************************** */

QCheckBox {
spacing: 5px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
font: bold 14px;
padding: 0px 0px 0px 5px;
}

QCheckBox::indicator {
width: 25px;
height: 25px;
border-width: 1px;
border-radius: 5px;
border-color: black;
border-style: outset;

background: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.689, fx:0.504926, fy:0.5, stop:0 rgba(253, 172, 0, 255), stop:0.528302 rgba(253, 173, 0, 255), stop:0.669951 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));


 }


 QCheckBox::indicator:unchecked {
     image: none;
 }


 QCheckBox::indicator:unchecked:pressed {
     image: url(:/Images/ok.png);
 }

 QCheckBox::indicator:checked {
     image: url(:/Images/ok.png);
 }


 QCheckBox::indicator:checked:pressed {
     image: none;
 }



/* *********************************************************** */

 QProgressBar {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
text-align: center;
font: bold 14px;
 }

 QProgressBar::chunk {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
width: 5px;
margin: 0.5px;
}

/* *********************************************************** */

QTabWidget::pane 
{
     border-top: 2px solid #C2C7CB;
 }

QTabWidget::tab-bar {
left: 2px; 
border-style: none;
 }

 QTabBar::tab {
border: black;
border-style: outset;
border-width: 1px;
font: 30px;
border-top-left-radius: 5px;
border-top-right-radius: 5px;
min-width: 150px;
min-height:40px;
padding: 2px;
color: white;

 }


QTabBar::tab:selected {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
 }

QTabBar::tab:!selected {
background:  qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
 }

/* ***********************************************************/ 

QListView {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
border-width: 1px;
border-radius: 5px;
border-color: black;
border-style: outset;
font: bold 24px;
padding: 8 8px;
color: white;
 }

 QListView::item {
background-color: rgba(0,0,0,0);
 }

 QListView::item:selected {
 background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
 }

 QListView::item:selected:!active {
 background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
 }

 QListView::item:selected:active {
 background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
 }

 QListView::item:hover {
     color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
 }

/* *********************************************************** */

/* *********************************************************** */ 

QTableWidget {
show-decoration-selected: 1; /* make the selection span the entire width of the view */
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(119, 130, 137, 255), stop:1 rgba(53, 57, 60, 255));
border-width: 1px;
border-radius: 5px;
border-color: black;
border-style: outset;
font: bold 24px;
color: white;
padding: 8 8px;
background-attachment: fixed;
}

QTableView QTableCornerButton::section {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
border: 1px outset black;
}

QTableWidget::item {
background-color: rgba(119, 130, 137, 255);
}


QTableWidget::item:selected {
border: 1px solid #6a6ea9;
}

QTableWidget::item:selected:!active {
background: rgba(119, 130, 137, 255);
border: 1px solid #6a6ea9;
}

QTableWidget::item:selected:active {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 172, 0, 255), stop:1 rgba(195, 95, 0, 255));
}

QTableWidget::item:hover {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #FAFBFE, stop: 1 #DCDEF1);
}

QHeaderView::section {
background-color:  qlineargradient(x1:0, y1:0, x2:0, y2:1,
stop:0 #616161, stop: 0.5 #505050,
stop: 0.6 #434343, stop:1 #656565);
color: white;
padding-left: 3px;
border-left:  1px solid black;/*1px solid #6c6c6c;*/
border-right:1px solid black;
border-top: none;
border-bottom: 1px solid black;

}

/* *********************************************************** */

QScrollBar:horizontal {
border: 1px solid black;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(26, 26, 26, 255), stop:1 rgba(61, 61, 61, 255));
height: 15px;
margin: 0px 0px 0 0px;
border-radius: 5px;
}

QScrollBar::handle:horizontal {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 170, 0, 255), stop:1 rgba(195, 95, 0, 255));
min-width: 10px;
border-radius: 5px;
}

QScrollBar::add-line:horizontal {
background: blue;
width: 0px;
subcontrol-position: bottom right;
subcontrol-origin: margin;
border: 0px solid black;
border-radius: 5px;
}

QScrollBar::sub-line:horizontal {
background: blue;
width: 0px;
subcontrol-position: bottom left;
subcontrol-origin: margin;
border:0px solid black;
border-radius: 5px;
}

QScrollBar:left-arrow:horizontal, QScrollBar::right-arrow:horizontal {
width: 5px;
height: 5px;
background: solid;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
background: none;

}
/* *********************************************************** */

QScrollBar:vertical {
border: 1px solid black;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(26, 26, 26, 255), stop:1 rgba(61, 61, 61, 255));;
height: 15px;
margin: 0px 0px 0 0px;
border-radius: 5px;
}

QScrollBar::handle:vertical {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(253, 170, 0, 255), stop:1 rgba(195, 95, 0, 255));
min-width: 10px;
border-radius: 5px;
}

QScrollBar::add-line:vertical {
background: blue;
width: 0px;
subcontrol-position: bottom right;
subcontrol-origin: margin;
border: 0px solid black;
border-radius: 5px;
}

QScrollBar::sub-line:vertical {
background: blue;
width: 0px;
subcontrol-position: bottom left;
subcontrol-origin: margin;
border:0px solid black;
border-radius: 5px;
}

QScrollBar:left-arrow:vertical, QScrollBar::right-arrow:vertical {
width: 5px;
height: 5px;
background: solid;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background: none;

}

/* *********************************************************** */





/* *********************************************************** */



/* *********************************************************** */







</string>
  </property>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Close application</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>79</y>
     <width>761</width>
     <height>391</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label0">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>This is page 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NextScreen0">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>230</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddScreen0">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>160</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>171</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>14</number>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>201</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>331</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>545</x>
       <y>0</y>
       <width>211</width>
       <height>251</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>180</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>431</x>
       <y>230</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Agree</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>431</x>
       <y>260</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>180</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_2</string>
     </attribute>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>260</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_3</string>
     </attribute>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>300</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_3</string>
     </attribute>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>340</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup_3</string>
     </attribute>
    </widget>
    <zorder>radioButton_2</zorder>
    <zorder>label0</zorder>
    <zorder>NextScreen0</zorder>
    <zorder>AddScreen0</zorder>
    <zorder>listWidget</zorder>
    <zorder>calendarWidget</zorder>
    <zorder>tableWidget</zorder>
    <zorder>treeWidget</zorder>
    <zorder>radioButton</zorder>
    <zorder>radioButton_3</zorder>
    <zorder>checkBox</zorder>
    <zorder>checkBox_2</zorder>
    <zorder>radioButton_4</zorder>
    <zorder>radioButton_6</zorder>
    <zorder>commandLinkButton</zorder>
    <zorder>commandLinkButton_2</zorder>
    <zorder>commandLinkButton_3</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>310</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>Application</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
 </buttongroups>
</ui>
